code/game/machinery/computer/dna_console.dm

```diff
@@ -40,6 +40,16 @@
 #define GENETIC_DAMAGE_PULSE_UNIQUE_IDENTITY "ui"
 #define GENETIC_DAMAGE_PULSE_UNIQUE_FEATURES "uf"
 
+/// Input from tgui interface. X the gene out.
+#define CLEAR_GENE 0
+/// Input from tgui interface. Progress to the next gene.
+#define NEXT_GENE 1
+/// Input from tgui interface. Progress to previous gene.
+#define PREV_GENE 2
+
+/// List of genes in sequence for iterating.
+#define GENE_LETTERS list("A", "T", "C", "G");
```

Comment:
I don't think this should be a define at all, you only use it in one place, and you name it `gene_letters` again directly before

Reply:
~~list defines are a bad pattern anyway~~

Reply:
Moved to the static list definition.